ivy/func_wrapper.py

```diff
@@ -38,6 +38,10 @@
     "handle_nans",
 ]
 
+CYTHON_WRAPPERS = [
```

Comment:
is there a way we can avoid hardcoding a list here? Like is there a way we can just infer the wrappers based on the .so files available in the system and their names?

Reply:
That is the intended way, the hardcoded list is only here for now for ease of testing. I'll update it soon. The logic for available `.so` files is already implemented, but not all of them work hence the hard-coded list.